hepdata/modules/dashboard/api.py

```diff
@@ -247,3 +251,31 @@ def get_pending_invitations_for_user(user):
              'role': invite.role, 'coordinator': coordinator})
 
     return result
+
+
+def get_dashboard_current_user(current_user):
+    """Gets the user to display in the dashboard.
+
+    For non-admin users this will just return the current user.
+    For admin users, if they have chosen to view the dashboard as another user,
+    this will return that user.
+
+    :param invenio_accounts.models.User current_user: Currently logged-in user, e.g. flask_login.current_user
```

Comment:
`invenio_accounts.models.User` is the param type. The docblock generates html like:

<dl class="py function">
<dt class="sig sig-object py" id="hepdata.modules.dashboard.api.get_dashboard_current_user">
<span class="sig-prename descclassname"><span class="pre">hepdata.modules.dashboard.api.</span></span><span class="sig-name descname"><span class="pre">get_dashboard_current_user</span></span><span class="sig-paren">(</span><em class="sig-param"><span class="n"><span class="pre">current_user</span></span></em><span class="sig-paren">)</span><a class="reference internal" href="../_modules/hepdata/modules/dashboard/api.html#get_dashboard_current_user"><span class="viewcode-link"><span class="pre">[source]</span></span></a><a class="headerlink" href="#hepdata.modules.dashboard.api.get_dashboard_current_user" title="Permalink to this definition">¶</a></dt>
<dd><p>Gets the user to display in the dashboard.</p>
<p>For non-admin users this will just return the current user.
For admin users, if they have chosen to view the dashboard as another user,
this will return that user.</p>
<dl class="field-list simple">
<dt class="field-odd">Parameters</dt>
<dd class="field-odd"><p><strong>current_user</strong> (<em>invenio_accounts.models.User</em>) – Currently logged-in user, e.g. flask_login.current_user</p>
</dd>
<dt class="field-even">Returns</dt>
<dd class="field-even"><p>User to display in the dashboard</p>
</dd>
<dt class="field-odd">Return type</dt>
<dd class="field-odd"><p>invenio_accounts.models.User</p>
</dd>
</dl>
</dd></dl>

Let me know if you'd prefer me to remove it though.

Reply:
My mistake, I thought it was an accidental cut-and-paste from two lines below.  I'll merge now.